release-notes/major12/12.2.0/valtimo-frontend-libraries.md

```diff
@@ -4,60 +4,49 @@
+  Various dependencies in the libraries have been upgraded for security reasons.
 
-* **New feature2**
+* **Cypress removed from codebase**
 
-  Description of the new feature goes here.
-  Also link to the page that expains the feature in greater detail.
+  The infrastructure to implement Cypress end-to-end tests has been removed from the front-end libraries codebase,
+  since it was never used; integration testing is done using Playwright instead.
 
+* **Value path selector**
 
-## Bugfixes
-
-The following bugs were fixed:
-
-* **Bug1**
-
-  Description of what the issue was.
-
-* **Bug2**
+  In many configuration forms in the application, manual path inputs are used. For example, an admin fills in
+  `case:createdOn` for a search field configuration. A component has now been implemented to allow for the selection
+  of possible values for these fields using a dropdown. A manual input option is still available in these places by
+  switching a toggle. 
 
-  Description of what the issue was.
+  In this version, this value path selector has been implemented  for configuring task and case search fields, case list
+  columns, and for selecting the default value for Form.io components inside the form builder. The latter is available
+  under a Valtimo tab in configuration modals for each Form.io component.
 
-## Breaking changes
-
-The following breaking changes were introduced:
-
-* **Breaking change1**
+## Bugfixes
 
-* **Breaking change2**
+The following bugs were fixed:
 
-Instructions on how to migrate to this version of Valtimo can be found [here](migration.md).
+* **Form builder stability**
 
-## Deprecations
+  On the admin form management page, the form builder sometimes would not allow form components to be saved. This issue
+  should be resolved.
 
-The following was deprecated:
+* **Custom form.io component not rendering consistently**
 
-* **Deprecation1**
+  A fix has been implemented for custom Form.io components not rendering correctly in certain situations. To implement
+  this fix, please refer to the [migration notes of this release](migration.md).
 
-  X was deprecated and is replaced with Y.
-* **Deprecation2**
+* **Documenten API metadata modal in Form.io forms**
 
-  X was deprecated and is replaced with Y.
+  The documenten API metadata modal for the Documenten API uploader custom component for Form.io now works properly.
 
-Instructions on how to migrate to this version of Valtimo can be found [here](migration.md).
 
 ## Known issues
 
 This version has the following known issues:
 
-* **Issue1**
-    * Discovered in version x.x.x
-    * Describe what can be done to work around the issue
-
-* **Issue2**
-    * Discovered in version x.x.x
-    * Describe what can be done to work around the issue
+* **Camera feature for custom Form.io upload components might not work properly**
+    * Discovered in version 12.1.0
+    * If you encounter this issue, the camera feature can be disabled in the Form.io form configuration.
```

Comment:
Since this issue is solved in 12.2.0, it shouldn't be in this list. Maybe move to 12.1.0?